ROMFS/px4fmu_common/init.d/13000_generic_vtol_standard

```diff
@@ -24,7 +24,7 @@ if [ $AUTOCNF == yes ]
 then
 	param set VT_TYPE 2
 	param set VT_MOT_COUNT 4
-	param set PWM_AUX_DIS5 950
+	param set PWM_AUX_DISARMED 950
```

Comment:
I'm not sure we're going in the right direction here: The DIS5 value applied only to the pusher motor (does it still?). We also need to make sure these strings are not copied into the ENV vars of each new application - I've created a separate issue for this: https://github.com/PX4/Firmware/issues/10164

Reply:
The original code in this file was:
```

set PWM_ACHDIS 5
set PWM_AUX_DISARMED 950
```
So this file requires both if `PWM_ACHDIS` is being replaced by `PWM_AUX_DIS5`:
```
param set PWM_AUX_DIS5 950
param set PWM_AUX_DISARMED 950
```
I'll document the other files similarly so that the remaining cases are easier to scrutinize.

Reply:
Are you sure about that? Because IIRC then that set channel 5 to 950. And if I read your cleaned version (much appreciated by the way! The work you're doing is super important!) then we're setting now all aux channels to a disarmed value of 950.

The significance of that is that the default for disarmed is no pulse at all, so now having a disarmed pulse is a significant change and can interfere with things like camera triggers.

So to me the 1:1 equivalent change would be `param set PWM_AUX_DIS5 950` and not setting `PWM_AUX_DISARMED` at all - but again, I have not traced the full execution and I might be wrong.

Reply:
Thanks again @LorenzMeier , I've pushed up changes to this PR that keep your suggested 1:1, and adds an initialization `PWM_AUX_RATE p:PWM_AUX_RATE` and:
```
if [ $PWM_AUX_DISARMED != none ]
then
	pwm disarmed -c ${PWM_AUX_OUT} -p ${PWM_AUX_DISARMED} -d ${OUTPUT_AUX_DEV}
fi
```